### PR TITLE
toJSON consistency

### PR DESCRIPTION
**Pull request by Khaaz**
fix toJSON missing properties in some of the structures.
I might have missed some

### CODE REVIEW

**Comment by apacheli** [comment]
Little nitpick, but would it be worth putting the JSON properties in alphabetical order? Considering one of the last PRs made sorting changes.

File: lib/structures/Guild.js [hunk 1]
```diff
@@ -899,34 +899,41 @@ class Guild extends Base {
...
+            "preferredLocale",
+            "description",
+            "maxMembers",
+            "publicUpdatesChannelID",
+            "rulesChannelID",
+            "maxVideoChannelUsers",
+            "widgetEnabled",
+            "widgetChannelID",
+            "approximateMemberCount",
+            "approximatePresenceCount",
```
Comment (bsian03) [inline lib/structures/Guild.js]: Keep this alphabetical

Comment (bsian03) [reply lib/structures/Guild.js]: Also you're missing `shard`, `voiceStates`, `applicationID`, `autoRemoved`, `emojiCount`, `primaryCategoryID`, `primaryCategory`, `categories`, `keywords`, `pendingVoiceStates`, `discoverySplash`, `systemChannelFlags`, `vanityURL`

File: lib/structures/Message.js [hunk 2]
```diff
@@ -415,6 +415,9 @@ class Message extends Base {
             "embeds",
             "hit",
             "mentionEveryone",
+            "messageReference",
+            "flags",
+            "webhookID",
```
Comment (bsian03) [inline lib/structures/Message.js]: alphabetical order

Comment (bsian03) [reply lib/structures/Message.js]: Also missing `channel`, `guildID`, `referencedMessage`, `member`, `activity`, `application`

**Review by bsian03 — CHANGES_REQUESTED** [review]

File: lib/structures/Guild.js [hunk 3]
```diff
@@ -901,32 +901,46 @@ class Guild extends Base {
         return super.toJSON([
```
Comment (bsian03) [inline lib/structures/Guild.js]: `shard`, `voiceStates`, `categories`, `pendingVoiceStates`, `discoverySplash`, `vanityURL` still missing

**Review by bsian03 — CHANGES_REQUESTED** [review]

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**